### ISSUE TITLE
NTR:visually-mediated background adaptation

### ISSUE THREAD

**Issue by cerivs** [geneontology/go-ontology#20524]
A process  in  which  organisms  lighten  their  pigmentation  in response to a bright environment and darken it in a low-light environment.

PMID: 10493760

establishment of pigment granule localization  GO:0051905 is part of the process

There is a visual perception component.

**Comment by krchristie** [comment]
Hi @cerivs 

It seems to me that background adaptation isn't necessarily mediated visually. It's known that there are opsins that respond to light in ways that don't contribute to vision, for example melanopsin and skin-expressed neuropsin both have roles in regulating circadian rhythms. This quote from the paper you cite also suggests that the background adaptation response does not correlate completely with blindness:

> Thirteen mutants in our collection that fail to adapt to the background are also clearly blind (Tables 3, 4). Other mutants with apparent failure to aggregate pigment display normal OKR and OMR, most prominently fata morgana, submarine,microps, and wavy. 

I'm wondering if it would be better to have a broader term name that doesn't specify "visually-mediated". The definition you suggest wouldn't need modification as it already doesn't mention any connection to vision.

So, I'd propose this term stanza:

**Name**: Light-intensity mediated background adaptation
**Definition**: A process in which organisms lighten their pigmentation in response to a bright environment and darken it in a low-light environment. [PMID:10493760, GOC:cvs, GOC:krc]
**is_a**: GO:0009642 ! response to light intensity
**has_part**: establishment of pigment granule localization GO:0051905

Does that sound OK to you?

Thanks,

-Karen

**Comment by krchristie** [comment]
emailed @cerivs on 1/5/2021

**Comment by cerivs** [comment]
It will probably work. 
Zebrafish researchers are fairly consistent in calling the process a visual process; From PMID: 19549326
Visually mediated background adaptation (VBA)for disruption in neuropeptide transcription. VBA is a well characterized  neuroendocrine  reflex  that  occurs  in  fish,reptiles, and amphibians (reviewed in [26]). When placed on a dark, light-absorbing background, pituitary melanotropes  release  α-melanocyte-stimulating  hormone;  this neuropeptide mediates dispersion of black pigment granules  in  dermal  melanophores  and  causes  a  darkening  of skin  color  in  fish.  Conversely,  when  placed  on  a  light background,  a  drop  in  α-melanocyte-stimulating  hormone  levels  results  in  the  aggregation  of  melanin  and  a loss  of  this  'dark'  phenotype  or  an  overall  blanching. "

The cited article is https://pubmed.ncbi.nlm.nih.gov/17389778/  which is behind a paywall.

**Comment by krchristie** [comment]
To me it looks like you need more than one term. Here's a fairly recent review. The abstract has a nice summary. I've bolded some parts and added numbers to highlight their listing of the multiple mechanisms that affect background adaptation.

Bertolesi GE, McFarlane S. **Seeing the light to change colour: An evolutionary perspective on the role of melanopsin in neuroendocrine circuits regulating light-mediated skin pigmentation.** Pigment Cell Melanoma Res. 2018 May;31(3):354-373. PMID:29239123. https://pubmed-ncbi-nlm-nih-gov.ezproxy.jax.org/29239123/

**Abstract**: Melanopsin photopigments, Opn4x and Opn4m, were evolutionary selected to "see the light" in systems that regulate skin colour change. In this review, we analyse the roles of melanopsins, and how critical evolutionary developments, including the requirement for thermoregulation and ultraviolet protection, the emergence of a background adaptation mechanism in land-dwelling amphibian ancestors and the loss of a photosensitive pineal gland in mammals, may have helped sculpt the mechanisms that regulate light-controlled skin pigmentation. These mechanisms include **[1] melanopsin in skin pigment cells directly inducing skin darkening for thermoregulation/ultraviolet protection**; **[2] melanopsin-expressing eye cells controlling neuroendocrine circuits to mediate background adaptation in amphibians in response to surface-reflected light**; and **[3] pineal gland secretion of melatonin phased to environmental illuminance to regulate circadian and seasonal variation in skin colour**, a process initiated by melanopsin-expressing eye cells in mammals, and by as yet unknown non-visual opsins in the pineal gland of non-mammals. 

Bertolesi GE, McFarlane S. **Melanin-concentrating hormone like and somatolactin. A teleost-specific hypothalamic-hypophyseal axis system linking physiological and morphological pigmentation.** Pigment Cell Melanoma Res. 2020 Sep 8.  PMID:32898924.

**INTRODUCTION** - Altering skin colour in response to changes in background allows organisms to survive in a wide range of conditions, contributes to the invasion of novel habitats and helps to perpetuate the species. In organisms with a “light exposed integument” (fish, amphibians and reptiles), a sudden change in the background colour triggers two different biological processes to match skin colour to the surroundings (crypsis). First, an initial fast response, named skin physiological colour change (chromomotor change), involves the aggregation/dispersion of pigment‐containing organelles in pigmented skin cells (chromatophores); second, a much slower response called morphological colour change (chromogenic change) occurs with “long‐term” adaptation to the background and produces alterations in the amount or type of pigment and/or the number of skin chromatophores (Leclercq, Taylor, & Migaud, 2009; Sugimoto, 2002). 

So from this, it looks to me like you need multiple terms, a basic one and some more specific types. I feel pretty confident about these terms, though there may be need for more. 
-- background adaptation
--- visually-mediated background adaptation
--- integument-mediated background adaptation

The paper you cited in your original request clearly includes some mutants that do not appear to affect the visually-mediated form of background adaptation since they say "Other mutants with apparent failure to aggregate pigment display normal OKR and OMR, most prominently fata morgana, submarine,microps, and wavy."

**Comment by cerivs** [comment]
Those look good.  The paper originally cited by the author is one of the seminal screening papers so it described multiple different mutants and processes in a very cursory fashion.  The 3 terms you propose are great.

**Comment by krchristie** [comment]
Below are the three terms for background adaptation that I'm adding. Let me know if I missed anything.

I also felt that the term `background adaptation` needed a parent term of `regulation of pigmentation`. The most general existing regulation term for pigmentation was `regulation of developmental pigmentation`, so I created a new term for `regulation of pigmentation` also.

thanks, Karen

+[Term]
+id: GO:0120302
+name: background adaptation
+namespace: biological_process
+def: "Any process in which an organism changes its pigmentation (lightening in response to a brighter environment or darkening in response to a dimmer environment) in response to a change in light intensity." [GOC:cvs, GOC:krc, PMID:10493760, PMID:29239123, PMID:32898924]
+is_a: GO:0009642 ! response to light intensity
+created_by: krc
+creation_date: 2021-02-09T18:11:11Z
+
+[Term]
+id: GO:0120303
+name: visually-mediated background adaptation
+namespace: biological_process
+def: "Any process in which an organism changes its pigmentation (lightening in response to a brighter environment or darkening in response to a dimmer environment) in response to a change in light intensity detected by melanopsin-expressing eye cells." [GOC:cvs, GOC:krc, PMID:29239123, PMID:32898924]
+is_a: GO:0120302 ! background adaptation
+created_by: krc
+creation_date: 2021-02-09T18:35:11Z
+
+[Term]
+id: GO:0120304
+name: integument-mediated background adaptation
+namespace: biological_process
+def: "Any process in which an organism changes its pigmentation (lightening in response to a brighter environment or darkening in response to a dimmer environment) in response to a change in light intensity detected by light sensitive cells in the integument." [GOC:cvs, GOC:krc, PMID:29239123, PMID:32898924]
+is_a: GO:0120302 ! background adaptation
+created_by: krc
+creation_date: 2021-02-09T18:41:07Z



+[Term]
+id: GO:0120305
+name: regulation of pigmentation
+namespace: biological_process
+def: "Any process that modulates the frequency, rate or extent of the deposition or modulates the distribution of coloring matter in an organism." [GOC:krc]
+intersection_of: GO:0065007 ! biological regulation
+intersection_of: regulates GO:0043473 ! pigmentation
+created_by: krc
+creation_date: 2021-02-09T19:37:35Z

**Comment by cerivs** [comment]
Those look great.  Thanks for all the work you have done on this ticket, helping get the right terms not just the expedient terms.
